sdk/textanalytics/ai-text-analytics/package.json

```diff
@@ -6,7 +6,8 @@
   "version": "5.0.1",
   "keywords": [
     "node",
-    "azure",
+    "Azure",
```

Comment:
I believe there must be a reason where everything in this list is in lower case.
If the keyword based npm search does not care about casing, we should just update the linter rule to accept "azure" instead of "Azure".

Reply:
It looks like npm search does not care about casing, so I opened https://github.com/Azure/azure-sdk-for-js/pull/10359.